File: 프로그래머스/lv0/120893. 대문자와 소문자/대문자와 소문자.cs
using System;
using System.Text;

public class Solution {
    public string solution(string my_string) {

        var sb = new StringBuilder();
        for(int i = 0; i < my_string.Length; i++)
        {
            char c = my_string[i];
            if(char.IsUpper(c))
                sb.Append(char.ToLower(c));
            else
                sb.Append(char.ToUpper(c));
        }
        
        return sb.ToString();
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string) {
    string answer = "";
    
    for (char c : my_string)
    {
        if (c > 'Z') // 소문자
            answer += (c - 'a' + 'A');
        else
            answer += (c + 'a' - 'A');
    }
    
    return answer;
}